acm/src/main/java/org/literacybridge/acm/api/IDataRequestResult.java

```diff
@@ -12,30 +10,18 @@
  *
  */
 public interface IDataRequestResult {
-	/**
-	 * Returns the root category which should be used to walk
-	 * the tree of categories. The root category is a virtual root
-	 * and should generally not be displayed by the UI.
-	 */
-	public abstract Category getRootCategory();
+    /**
+     * Returns the facet count for the passed-in category.
```

Comment:
In my latest commit I removed the old interface IDataRequestService and class DataRequestService and changed the places that used it to use the new MetadataStore directly. Also, I removed IDataRequestResult and renamed DataRequestResult to SearchResult. I added documentation to SearchResult describing what 'Facet count' means (it's actually not only a Lucene concept)